src/GeneratorV2/Process/Processor.cs

```diff
@@ -944,7 +1015,7 @@ public void WritePrologue(IndentedTextWriter writer, NameTable nameTable)
             }
         }
 
-        public class GenCreateAndDeleteOverloader : IOverloader
+        class GenCreateAndDeleteOverloader : IOverloader
```

Comment:
I suspect this change will create some merge errors with #1284 as that PR moves these classes into seperate files.